README.md

```diff
@@ -5,9 +5,15 @@
 
 <!-- /!\ do not modify above this line -->
 
-# web
+Web addons for Odoo
+===================
```

Comment:
You probably want to put it in there too:
https://github.com/OCA/web/blob/14.0/.copier-answers.yml#L10

Let's also keep the `#` for title levels, it's easier to maintain.

Reply:
@yvaucher done